sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/util/PsiClassSearchHelper.kt

```diff
@@ -1,25 +1,54 @@
 package app.cash.sqldelight.intellij.util
 
+import app.cash.sqldelight.intellij.util.PsiClassSearchHelper.ImportableType.JavaType
+import app.cash.sqldelight.intellij.util.PsiClassSearchHelper.ImportableType.KotlinType
 import com.intellij.openapi.project.Project
-import com.intellij.psi.JavaPsiFacade
 import com.intellij.psi.PsiClass
 import com.intellij.psi.search.GlobalSearchScope
 import com.intellij.psi.search.PsiShortNamesCache
-import org.jetbrains.kotlin.idea.refactoring.memberInfo.qualifiedClassNameForRendering
 import org.jetbrains.kotlin.idea.stubindex.KotlinClassShortNameIndex
+import org.jetbrains.kotlin.psi.KtClassOrObject
+import org.jetbrains.kotlin.psi.psiUtil.getChildrenOfType
+import javax.swing.Icon
 
-object PsiClassSearchHelper {
-
-  fun getClassesByShortName(shortName: String, project: Project, scope: GlobalSearchScope): List<PsiClass> {
-    val javaPsiFacade = JavaPsiFacade.getInstance(project)
+internal object PsiClassSearchHelper {
 
+  fun getClassesByShortName(shortName: String, project: Project, scope: GlobalSearchScope): List<ImportableType> {
```

Comment:
I guess we need typealiases as well. There's `KotlinTypeAliasShortNameIndex` for that